### PR TITLE
Outdated browser message

### PR DESCRIPTION
**Pull request by stuartsan**
Shows this to users whose browser doesn't support flexbox/webgl:

image

### CODE REVIEW

File: src/assets/styles/outdated_browser.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    &:link, &:visited, &:hover {
+      color: $aclu-blue;
+    }
+  }
+}
+
+.no-flexbox, .no-webgl {
+  .outdated-browser {
+    display: block;
+  }
```
Comment (ryanhobsonsmith) [inline src/assets/styles/outdated_browser.scss:27]: I get a weird flickering gray line on my comp (prob due to z-fighting between this overlay and mapbox). Simple solution was to just set display: none on the ```.event-map``` here as well. FYI, I got this after just forcing display:block on my mb-pro in chrome, might not be an actually issue on truly outdated browsers that can't load webgl or flexbox at all. However, traveling and don't have an easy ability to QA that at the moment.
screen shot 2017-04-26 at 7 14 02 pm

File: src/templates/OutdatedBrowser.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="outdated-browser">
+  <p>
+    Sorry, you are using an outdated browser.
+  </p>
+  <p>
+    Please <a href="http://outdatedbrowser.com/en">upgrade to a newer version</a>
```
Comment (ryanhobsonsmith) [inline src/templates/OutdatedBrowser.html:9]: Also, probably not much of an issue, but realized that this site isn't the greatest for mobile browsers... not sure there is actually much to be done in that case though.

Comment (stuartsan) [reply src/templates/OutdatedBrowser.html:9]: Oh yeah, good point, hrm. Yeah agree, not sure what we can do about that for the moment...

File: src/assets/styles/outdated_browser.scss (hunk 1 above)
Comment (stuartsan) [reply src/assets/styles/outdated_browser.scss:27]: I am unable to repro in chrome poking around, or IE9/10 (both unsupported browsers). Going to leave as-is for now.

**Comment by stuartsan** [comment]
I updated the unsupported browser message to show the aclu/people power logo. I think it looks quite a bit better...see initial comment for pic.

**Comment by stuartsan** [comment]
Going to merge this and we can iterate as needed -- for the moment we just need something :grin:

**Pull request merged**